src/adapter/src/coord/ddl.rs

```diff
@@ -572,6 +595,16 @@ impl Coordinator {
                 }
                 self.drop_compute_sinks(sink_ids);
             }
+            if !copy_to_sinks_to_drop.is_empty() {
+                let mut sink_ids = Vec::new();
+                for (conn_id, sink_id) in copy_to_sinks_to_drop {
+                    if let Some(conn_meta) = self.active_conns.get_mut(&conn_id) {
+                        conn_meta.drop_sinks.retain(|id| *id != sink_id);
```

Comment:
Like subscribe, should we emit a notice for a dropped copy_to?

Reply:
This got refactored a lot in https://github.com/MaterializeInc/materialize/pull/25179, which is not sending a notice anymore. Will revisit after that's merged.